filebeat/inputsource/inputsource.go

```diff
@@ -0,0 +1 @@
+package inputsource
```

Comment:
Not a fan of the naming here but as it's not user facing we can figure out the "correct" name when we have multiple use cases. This will make it easier.

Reply:
More I think about it.. using _source_ instead of `inputsource` would make sense. @ruflin @andrewkroh

Reply:
If we call this one source, what should we call this one then? https://github.com/elastic/beats/blob/master/filebeat/harvester/source.go

I suggest to get the PR with this name in an iterate on top of it. There will be quite a few more renames I assume.